crates/common/src/preprocessor/deps.rs

```diff
@@ -234,7 +234,7 @@ impl<'hir> Visit<'hir> for BytecodeDependencyCollector<'hir> {
                         for &var in clause.args {
                             self.visit_nested_var(var)?;
                         }
-                        for stmt in clause.block {
+                        for stmt in clause.block.stmts {
```

Comment:
cc @grandizzy this is a change in `=0.1.4` of Solar (required for the `foundry-compilers` / `foundry-block-explorers` bump), I believe this fix to be correct but please double check